src/Loader/FilesystemLoader.php

```diff
@@ -170,7 +170,7 @@ public function exists($name)
         }
 
         try {
-            return false !== $this->findTemplate($name, false);
+            return false !== $this->findTemplate($name);
```

Comment:
Is this intentional? There is logic explicitly for this: `$throw = \func_num_args() > 1 ? func_get_arg(1) : true;`

Reply:
reverting